compiler/stable_mir/src/mir/body.rs

```diff
@@ -913,10 +913,16 @@ pub enum Mutability {
     Mut,
 }
 
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub enum Unsafe {
```

Comment:
Can you please add documentation to these two enumerations? It isn't clear to me why we need two different enumerations here.